tutorials/tour/basics.md

```diff
@@ -37,51 +37,52 @@ println("Hello," + " world!") // Hello, world!
 
 ### Values
 
-You can name results of expressions by using the `val` keyword.
+You can name results of expressions with the `val` keyword.
 
-```
+```tut
 val x = 1 + 1
-println(x) // 2
+println(x * x) // 4
```

Comment:
I wanted the example to have a _reason_ to use `val`, so we are illustrating not only the syntax, but also the use. With my change, introducing `x` now avoids repeating a computation (namely, the adding of 1 and 1).

Maybe I'm overthinking this. I don't have a strong feeling about it.